edge/discussion_materials/onnx-edge-scope and profiling.md

```diff
@@ -0,0 +1,75 @@
+
+## ONNX Edge profile
+
+The work of the ONNX Edge profile includes:
+
+1. Attributes / characteristics: Power, Size, accuracy, speed, Security, etc.
+
+2. ONNX operations subset
+
+3. Other ONNX related limitations 
+
+Work to define strict subsets of the operator sets which apply to edge devices which avoid complex operations imposed by limitations on edge devices with high computational complexity. Exclude use-cases not applicable to the edge (e.g. large scale training scenarios)
+
+
+###Metrics for ONNX Edge
+
+#### Accuracy
+
+The ONNX model zoo collected many wide-used models for image classification, face recognition and image segmentation. While the accuracy number varies, the top-5 accuracy seems acceptable with VGG, resent and mobile net are all above or close to 90%. Accuracy is critical for scenarios like smart city, face recognition and smart monitoring. If put these models in edge device in actual use, primary work is to keep the accuracy still in a decent level while compressing the model and changing ops to low-bit version. Detailed work can be achieved cooperating with the quantization WG.
+
+#### Size
+
+In ONNX 1.4 release, support for large models (larger than 2GB) and store the data externally is added as a new feature. But in edge device, we do not want such huge models running and consuming the precious memory. There will always be compromises between performance, speed, cost and consumption. There are many ways to compress our models while keeping the accuracy like changing to quantized ops, model compression etc. ONNX Edge WG works to define the compression method for ONNX model and to develope quantized ops.
+
+#### Speed
+
+In many scenarios, fast processing speed and low latency are required. Vendor-specific runtime optimization are encouraged while we should also keep our optimization work going. works on the speed over laps with some of the work on the bandwidth, since a broader bandwidth always provide less latency thus improving overall speed.
+
+#### Power consumption
+
+Power consumption is critical for an edge device. 
+
+//MLPerf community is a broad ML benchmark suite for measuring performance of ML software frameworks, ML hardware accelerators, and ML cloud platforms. In MLPerf Edge Inference WG, power consumption is outlined as a metrics to be measured thus I suggest that our community can cooperate with MLPerf community to improve our power consumption performance. MLPerf Edge Inference Power/Energy [4] is the official document of MLPerf on how to handle the measurement and benchmarking for the power/ energy consumption. I suggest we look into the doc and cooperate with the MLPerf community to improve our power/ energy consumption performance based on the Spec mentioned.//
+
+#### Security
+
+#### Bandwidth
```

Comment:
Not sure we need Bandwidth as attribute, as it may be implicit to "Data Locality" and "Latency"